modules/op-changing-default-service-account.adoc

```diff
@@ -0,0 +1,25 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: CONCEPT
+[id="op-changing-default-service-account_{context}"]
+= Changing the default service account
+
+You can change the default service account by editing the `default-service-account` field in the `.spec.pipeline` and `.spec.trigger` specifications. The default service account name is `pipeline`.
+
+.Example
```

Comment:
Could you please check all the examples in this assembly follow the same syntax. Go through the preview to check all example instances
The syntax is
.Example: A brief description

I found inconsistency like some examples do not have any description and others follow a different synatx (like Example of auto-pruning annotations)